pages/Documents/Reporting/OutboundConfigurationAPI/overview.md

```diff
@@ -0,0 +1,400 @@
+---
+pagename: Overview
+sitesection: Documents
+categoryname: Reporting
+documentname: Outbound Configuration API
+permalink: outbound-configuration-api-overview.html
+root-link: true
+indicator: Proactive Messaging
+redirect_from:
+ - proactive-messaging-reporting-api.html
+---
+
+### Introduction
+The Outbound configuration API provides a complete list of API handoffs created in the Proactive Web Tool. This API allows 1) Fetching all handoffs configurations 2) Fetching all handoff configurations filtered by a specific messaging channel (sms, whats app, apple business chat, in-app, google rcs etc)
```

Comment:
please fix: "whats app", "google",(use sentence case) please fix and check here
[Please use ](https://github.com/LivePersonInc/developer-center#notes-on-content-and-code)

Reply:
Done